packages/legacy/core/App/contexts/activity.tsx

```diff
@@ -33,17 +33,6 @@ export const ActivityProvider: React.FC<PropsWithChildren> = ({ children }) => {
   const inactivityTimeoutRef = useRef<NodeJS.Timeout | null>(null)
   const prevAppStateStatusRef = useRef(AppState.currentState)
   const [appStateStatus, setAppStateStatus] = useState<AppStateStatus>(AppState.currentState)
-  const panResponder = useRef(
-    PanResponder.create({
-      onStartShouldSetPanResponderCapture: () => {
-        // some user interaction detected, reset timeout
-        resetInactivityTimeout(timeoutInMilliseconds.current)
```

Comment:
Because it was a ref, only the first `resetInactivityTimeout` function was used which always had `didAuthenticate` as `false`. The new `useMemo` watches for changes to that function